### ISSUE TITLE
my training problem with yolov3

### ISSUE THREAD

**Issue by GYC1996** [AlexeyAB/darknet#2578]
I use yolov3 to train my dataset. With more iterations, I found the training loss decreasing, but my map value getting higher.I don't know whether it is a over-fitting problems?here is my picture.
![TIM截图20190312112753](https://user-images.githubusercontent.com/38093100/54173184-d3882980-44bb-11e9-8a72-80ec9d49baf7.png)
![TIM截图](https://user-images.githubusercontent.com/38093100/54173217-fb778d00-44bb-11e9-9c1d-388e20d7d6e0.png)

**Comment by GYC1996** [comment]
my map is calculated in my valid dataset

**Comment by arkjiang** [comment]
mAP is mean average precision. the higher the better. 
on the other hand, the lower model loss the better.
You shall pick the weight of around 10000 inter according to your mAP plot

**Comment by GYC1996** [comment]
I found I made a mistake for describing my problem above. 
map value getting lower and the trainging loss gitting lower with the increase of iters. I want to know whether it is a over-fitting problem

**Comment by GYC1996** [comment]
> mAP is mean average precision. the higher the better.
> on the other hand, the lower model loss the better.
> You shall pick the weight of around 10000 inter according to your mAP plot

I konw. But I want to improve it. In the past time, I use yolov3 to trainging my dataset, often performing the higher map with the lower loss.

**Comment by arkjiang** [comment]
> I found I made a mistake for describing my problem above.
> map value getting lower and the trainging loss gitting lower with the increase of iters. I want to know whether it is a over-fitting problem

yes. its overfitting. should stop when validation set is no longer improving.
